src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Endpoint.cs

```diff
@@ -0,0 +1,228 @@
+﻿using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Public.API.Helpers;
+using FastEndpoints;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Alignments.Greek;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, Response>
+{
+    private const string GreekAlignmentSourceQuery = """
+        SELECT TOP 1 gnt.[Name]
+        FROM BibleVersionWords bvw
+        LEFT JOIN BibleVersionWordGroupWords bvwgw ON bvwgw.BibleVersionWordId = bvw.Id
+        LEFT JOIN NewTestamentAlignments nta ON nta.BibleVersionWordGroupId = bvwgw.BibleVersionWordGroupId
+        LEFT JOIN GreekNewTestamentWordGroupWords gntwgw ON gntwgw.GreekNewTestamentWordGroupId = nta.GreekNewTestamentWordGroupId
+        LEFT JOIN GreekNewTestamentWords gntw ON gntw.Id = gntwgw.GreekNewTestamentWordId
+        LEFT JOIN GreekNewTestaments gnt ON gntw.GreekNewTestamentId = gnt.Id
+        WHERE BibleId = @BibleId
+        """;
+
+    private const string BaseQuery = """
+        SELECT bvw.Id, bvw.WordIdentifier, bvw.[Text] AS EnglishWord, bvwgw.BibleVersionWordGroupId
+        FROM BibleVersionWords bvw
+        LEFT JOIN BibleVersionWordGroupWords bvwgw ON bvwgw.BibleVersionWordId = bvw.Id
+        WHERE BibleId = @BibleId AND bvw.WordIdentifier BETWEEN @LowerBounds AND @UpperBounds
+        ORDER BY bvw.WordIdentifier
+        """;
+
+    private const string GreekWordsQuery = """
+        SELECT bvw.Id AS EnglishWordId, gntw.Id AS GreekNewTestamentWordId, gw.[Text] AS GreekWord
+        FROM BibleVersionWords bvw
+        LEFT JOIN BibleVersionWordGroupWords bvwgw ON bvwgw.BibleVersionWordId = bvw.Id
+        LEFT JOIN NewTestamentAlignments nta ON nta.BibleVersionWordGroupId = bvwgw.BibleVersionWordGroupId
+        LEFT JOIN GreekNewTestamentWordGroupWords gntwgw ON gntwgw.GreekNewTestamentWordGroupId = nta.GreekNewTestamentWordGroupId
+        LEFT JOIN GreekNewTestamentWords gntw ON gntw.Id = gntwgw.GreekNewTestamentWordId
+        LEFT JOIN GreekWords gw ON gntw.GreekWordId = gw.Id
+        WHERE BibleId = @BibleId AND bvw.WordIdentifier BETWEEN @LowerBounds AND @UpperBounds AND gntw.Id IS NOT NULL
+        """;
```

Comment:
I'm having trouble with the performance of this query (see PR description).  This PR is blocked until I can address it.